README.md

```diff
@@ -181,29 +181,52 @@ it into the "Index pattern" field, then click the "Next step" button.
 Discover section.
 
 
-#### Using Docker
+#### Using Docker Locally
 
-1. Download, install, and launch [Docker]
+1. Download, install, and launch [Docker](https://www.docker.com/products/docker-desktop)
 
-1. Set up the Docker image
+1. Build and launch the Docker containers: `docker-compose up`
 
-  ```
-  $ bin/setup --docker
-  ```
+1. Copy necessary config files locally. This dir is mounted as a volume, so new files also appear in the container.
+
+```sh
+cp config/application.yml.default config/application.yml
```

Comment:
this is duplicated with `bin/setup` and `bin/fast_setup` -- WDYT about breaking out a new script that's like `bin/setup_local_links` or something and call into it from all the places?

Reply:
And I just discovered by breaking something that it's the same in `.circleci/config.yml` .... so if we don't do this now, I'll definitely look into it for layet

Reply:
Moved this stuff into `bin/docker_setup`